source/common/upstream/load_balancer_impl.h

```diff
@@ -388,6 +396,13 @@ class EdfLoadBalancerBase : public ZoneAwareLoadBalancerBase {
 
   // Scheduler for each valid HostsSource.
   absl::node_hash_map<HostsSource, Scheduler, HostsSourceHash> scheduler_;
+  // Slow start related configs
+  const envoy::config::cluster::v3::Cluster::CommonLbConfig::EndpointWarmingPolicy
+      endpoint_warming_policy;
+  const uint32_t slow_start_window;
+  TimeSource& time_source_;
+  absl::node_hash_set<HostSharedPtr> hosts_in_slow_start_;
```

Comment:
You should be able to use a flat_hash_set here.

Reply:
it's using `absl::btree_set` now